### PR TITLE
test: Integration Test for installing a template

### LINKED ISSUE #415: Community Templates: Integration Test for installing a template
### Background
We've written integration tests for `startTemplateInstall` on CommunityTemplates Index. Now it's time to test the installation process - making sure the install button is visible and that clicking it installs the template.

### Acceptance Criteria
The installation process is covered by some integration tests

### PR DESCRIPTION
**Pull request by zoesteinkamp**
Closes #415 

- [ ] [Well-formatted commit messages](https://www.conventionalcommits.org/en/v1.0.0-beta.3/)
- [ ] Rebased/mergeable
- [ ] Tests pass

### CODE REVIEW

**Comment by hercules-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 56 | :red_circle:
accessibility | 65 | :yellow_circle:
best-practices | 87 | :green_circle:
seo | 80 | :yellow_circle:
pwa | null | :question:

**Review by hoorayimhelping — APPROVED** [review]
One question regarding upgrading react, but outstanding work!

File: package.json [hunk 1]
```diff
@@ -187,7 +187,7 @@
     "react-dimensions": "^1.2.0",
     "react-dnd": "^9.3.2",
     "react-dnd-html5-backend": "^9.3.2",
-    "react-dom": "^16.8.2",
+    "react-dom": "16.9.0",
```
[the remark this answers is not in the document]
Comment (zoesteinkamp) [reply package.json:194]: When we added the await function, i got an error saying i needed to upgrade react-dom. I could try reverting this, and see if everything still works without it

**Comment by hoorayimhelping** [comment]
handled by https://github.com/influxdata/ui/pull/545

**Pull request closed without merging**